packages/lib/src/footer/Footer.tsx

```diff
@@ -117,23 +40,21 @@ const BottomContainer = styled.div`
     align-items: center;
   }
 
-  border-top: ${(props) =>
-    `${props.theme.bottomLinksDividerThickness} ${props.theme.bottomLinksDividerStyle} ${props.theme.bottomLinksDividerColor}`};
-  margin-top: 16px;
+  border-top: var(--border-width-s) var(--border-style-default) var(--border-color-primary-medium);
+  margin-top: var(--spacing-gap-m);
 `;
 
 const ChildComponents = styled.div`
-  min-height: 16px;
+  min-height: var(--height-xxs);
   overflow: hidden;
```

Comment:
we should consider adding a color token to be applied by default to the children.